app/views/statistics/index.html.erb

```diff
@@ -1,6 +1,10 @@
 <% title _("Statistics"), _('Statistics') %>
 <div id="statistics">
-<% charts.each do |ch| %>
-  <%= content_tag(:div, ch, :class=>"stats-well") %>
-<% end %>
+  <% @stats.each do |stat| %>
+    <div data-ajax-url=<%= stat.url %> class='stats-well' data-on-complete='refreshCharts'>
+      <div class='statistics-pie-placeholder'>
+        <h4 class='ca pie-title'> <span class='spinner spinner-xl spinner-inline'> </span> <%= stat.title %> </h4>
```

Comment:
I'll update to lg, but lg by itself looks bad on such a large space, do you
happen to have a better css alternative for me? :)

thanks